src/app/features/notification/notification.component.html

```diff
@@ -0,0 +1,97 @@
+<button
+  mat-icon-button
+  matTooltip="Notifications"
+  i18n-matTooltip="notifications toolbar icon tooltip"
+  [matMenuTriggerFor]="notificationList"
+>
+  <span
+    [matBadge]="
+      unreadNotifications?.length > 0 ? unreadNotifications?.length : null
+    "
+    matBadgeColor="accent"
+  >
+    <fa-icon class="white" icon="bell"></fa-icon>
+  </span>
+</button>
+
+<mat-menu #notificationList>
+  <div class="notification-panel" (click)="$event.stopPropagation()">
+    <!-- stopPropagation above to avoid closing the panel upon action button clicks -->
+
+    <div
+      class="flex-row justify-space-between align-center notification-panel-header"
+    >
+      <h2 class="notification-title" i18n>Notifications</h2>
+
+      <button
+        mat-icon-button
+        [matMenuTriggerFor]="settingsMenu"
+        #settingsMenuTrigger="matMenuTrigger"
+      >
+        <fa-icon icon="ellipsis"></fa-icon>
+      </button>
+    </div>
+
+    <!-- Tabs -->
+    <mat-tab-group [(selectedIndex)]="selectedTab">
+      <mat-tab label="All" i18n-label></mat-tab>
+      <mat-tab label="Unread" i18n-label></mat-tab>
+    </mat-tab-group>
+
+    <div class="flex-column gap-small">
```

Comment:
Can you design the divs in a way that we only have a scrollbar within the items and the title + tabs are always visible outside of the scroll box?

![image](https://github.com/user-attachments/assets/11f6dfb2-0edb-49d1-a67c-7a9c166ebeeb)

Reply:
Yes, we can do this. I have updated the code accordingly. but slight text is shown in the background at the top when we scroll the notification list.

Reply:
It's working properly now. When the user scrolls through the notification list, only the notification list will scroll, while the title and tabs remain fixed.

Reply:
somehow, if I scroll all the way to the bottom the top bar eventually scrolls out, although initially it remains fixed as intended, @Ayush8923

Reply:
@sleidig  I have fixed this. Now, when the user scrolls the notification list then the title and tabs remain fixed.

Reply:
your fixed caused a horizontal scrollbar for me and displayed the scroll across the whole menu rather than just the part that is actually scrollable. I made some further adjustments to fix this.